satpy/tests/writer_tests/cf_tests/test_attrs.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2017-2023 Satpy developers
+#
+# This file is part of satpy.
+#
+# satpy is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later
+# version.
+#
+# satpy is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# satpy.  If not, see <http://www.gnu.org/licenses/>.
+"""Tests for CF-compatible attributes encoding."""
+import datetime
+import json
+from collections import OrderedDict
+
+import numpy as np
+
+
+class TestCFAttributeEncoding:
+    """Test case for CF attribute encodings."""
+
+    def get_test_attrs(self):
+        """Create some dataset attributes for testing purpose.
+
+        Returns:
+            Attributes, encoded attributes, encoded and flattened attributes
+
+        """
+        # TODO: this is also used by test_da2cf
+        attrs = {'name': 'IR_108',
```

Comment:
This is used twice across the tests. 
Do we want to define that as some sort of `fake_attrs` to be imported across test files? Where should be defined?

Reply:
~Which tests use it?~ Ah I see your other comment. Extracting it to a separate function that can be imported seems fine.